pkg/nfd-topology-updater/nfd-topology-updater.go

```diff
@@ -337,6 +339,45 @@ func (w *nfdTopologyUpdater) updateNodeResourceTopology(zoneInfo v1alpha2.ZoneLi
 	return nil
 }
 
+func getCPUCoreAttribute(entry string) (v1alpha2.AttributeInfo, error) {
+	cpus, err := os.ReadFile(entry)
+	if err != nil {
+		klog.ErrorS(err, "error reading cpu entry file", "entry", entry)
+		return v1alpha2.AttributeInfo{}, err
```

Comment:
I'd suggest  to leave the logging to the caller
```suggestion
		return v1alpha2.AttributeInfo{}, err
```